docs(README.md): Add definition for `alpha` (#139)
d3/d3-force: README.md

<a name="simulation_alpha" href="#simulation_alpha">#</a> <i>simulation</i>.<b>alpha</b>([<i>alpha</i>]) [<>](https://github.com/d3/d3-force/blob/master/src/simulation.js#L98 "Source")

> The *alpha* is the current “heat” of the simulation. It decreases over time, as the simulation "cools down". When *alpha* reaches *alphaMin*, the simulation stops. See also [Simulation Restart](#simulation_restart)

Reviewer: ```suggestion
The *alpha* is the current “heat” of the simulation. It decreases over time, as the simulation “cools down”. When *alpha* reaches *alphaMin*, the simulation stops. See also [Simulation Restart](#simulation_restart).
```

> The *alpha* is the current “heat” of the simulation. It decreases over time, as the simulation "cools down". When *alpha* reaches *alphaMin*, the simulation stops. See also [Simulation Restart](#simulation_restart)

Reviewer: ```suggestion
*alpha* is the current “heat” of the simulation. It decreases over time, as the simulation “cools down”. When *alpha* reaches *alphaMin*, the simulation stops; see [*simulation*.restart](#simulation_restart).
```

> The *alpha* is the current “heat” of the simulation. It decreases over time, as the simulation "cools down". When *alpha* reaches *alphaMin*, the simulation stops. See also [Simulation Restart](#simulation_restart)

Reviewer: ```suggestion
*alpha* is roughly analogous to temperature in [simulated annealing](https://en.wikipedia.org/wiki/Simulated_annealing#Overview). It decreases over time as the simulation “cools down”. When *alpha* reaches *alphaMin*, the simulation stops; see [*simulation*.restart](#simulation_restart).
```
